pkg/cli/cmd/deploy/deploy_test.go

```diff
@@ -53,7 +54,7 @@ func Test_Validate(t *testing.T) {
 			},
 			ConfigureMocks: func(mocks radcli.ValidateMocks) {
 				mocks.ApplicationManagementClient.EXPECT().
-					GetEnvironment(gomock.Any(), radcli.TestEnvironmentName).
+					GetEnvironment(gomock.Any(), "/planes/radius/local/resourceGroups/test-resource-group/providers/Applications.Core/environments/test-environment").
```

Comment:
Should we have another test just with the name instead of the resource id?

Reply:
I did not add it since most of the tests use a name and not ID for testing..

Reply:
sorry, I just noticed I covered it. They are "rad deploy - valid with environment"  and "rad deploy - valid with env ID".